<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zLProjetClass</class>
 <widget class="QMainWindow" name="zLProjetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Détection et mesure de la zone de lyse</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../zlicon48.ico</normaloff>../../../../../zlicon48.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnOuvrir">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>500</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+O</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Liste des images</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxlsImg">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalcul">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>460</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculer la surface</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExporter">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exporter le résultat</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+S</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>201</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>471</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Cliquer sur le bouton Ouvrir pour ajouter une ou plusieurs images</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_Ouvrir"/>
    <addaction name="actionExporter_r_sultat"/>
    <addaction name="action_Quitter"/>
   </widget>
   <widget class="QMenu" name="menuAid_e">
    <property name="title">
     <string>Aid&amp;e</string>
    </property>
    <addaction name="actionAid_e"/>
    <addaction name="action_propos"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>&amp;Outils</string>
    </property>
    <addaction name="action_Calcul_la_surface"/>
    <addaction name="actionMettreAEchelle"/>
    <addaction name="action_Effacer_la_liste_des_images"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menuOutils"/>
   <addaction name="menuAid_e"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Ouvrir">
   <property name="text">
    <string>&amp;Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAid_e">
   <property name="text">
    <string>A&amp;ide</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_propos">
   <property name="text">
    <string>À &amp;propos</string>
   </property>
  </action>
  <action name="action_Calcul_la_surface">
   <property name="text">
    <string>&amp;Calculer la surface</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_Effacer_la_liste_des_images">
   <property name="text">
    <string>&amp;Effacer la liste des images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExporter_r_sultat">
   <property name="text">
    <string>Exporter le &amp;résultat</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionMettreAEchelle">
   <property name="text">
    <string>&amp;Mettre à l'échelle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="zlprojet.qrc"/>
 </resources>
 <connections/>
</ui>
